Documentation/rust/quick-start.rst

```diff
@@ -185,8 +185,19 @@ can be generated by the ``rust-analyzer`` Make target.
 Configuration
 -------------
 
-``Rust support`` (``CONFIG_RUST``) needs to be enabled in the ``General setup``
-menu. The option is only shown if a suitable Rust toolchain is found (see
+Generate ``.config``. Run::
+
+  make LLVM=1 xconfig
+
+Here you could use other options like ``menuconfig`` or ``oldconfig`` instead of ``xconfig``.
```

Comment:
In general, we are trying to avoid making the document a tutorial on the kernel (there are already docs on that in `Documentation/process`, e.g. https://docs.kernel.org/process/index.html), so I wouldn't mention this here.

In other words, the goal is not to teach how to build/configure the kernel, but to show what is different with respect to C.

Maybe what we should do is be more clear about this at the top of the document, and link to those other docs. Something like...

```
This document describes how to get started with kernel development in Rust.

If you are new to kernel development, please take a look at ... beforehand.
```

Reply:
(This comment is about explicitly mentioning the different `*config` targets -- GitHub is not too clear on that)

Reply:
good point :)

> In other words, the goal is not to teach how to build/configure the kernel, but to show what is different with respect to C.

Agree.

Also sure we could add some clarity on linking the process/index.html doc at the top. 

But my primary concern here is, we are very clear about using `LLVM=1` to build the kernel but not that clear about using `LLVM=1` when generating the build config ( I didn't even know that generating that config has anything to do with `LLVM=1` flag). This may not be clear even after reading `process/index.html` because there LLVM is mentioned as optional there

To address that concern, at minimum we could ask user to use `LLVM=1` as a flag during configuration (without mentioning about how to generate or anything, which I assume your concern of keeping this doc independent of "how" config is generated with different targets)

My alternate proposal:
1. Add reference to https://docs.kernel.org/process/index.html at the top of the doc like you suggested
2. And add a FAQ section explaining this exact issue I faced. Something like 
```
if you don't see "Rust Support" in the General Setup menu, then try using LLVM toolchain (`LLVM=1`) when building the configuraiton.
```
3. Instead of (2), we can also add the same note under "Configuration" section without explicit FAQ section.

Like to know what do you think.